userspace/libscap/engine/savefile/converter/table.h

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+#pragma once
+
+#include <converter/types.h>
+#include <libscap/scap_const.h>
+#include <driver/ppm_events_public.h>
+
+#include <unordered_map>
+
+static std::unordered_map<conversion_key, conversion_info> g_conversion_table = {};
```

Comment:
Do we really need this header?
Also, every time `table.h` gets included in a source file, we get a shiny new `g_conversion_table`; this is really error prone.
Instead i'd either:
* declare and define it under converter.cpp
* declare it extern here and define it in converter.cpp
* declare it extern here and define it in a new source file `table.c`

Reply:
Fair enough! I picked `declare it extern here and define it in a new source file table.c`